app/controllers/api/greetings_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::GreetingsController < ApplicationController
+  def index
+    greeting = Message.find(rand(1...5))
+
+    render json: greeting
+  end
+end
```

Comment:
- [ ] I am unable to see any random greeting in the front-end, I am getting the following message, on the back-end, kindly fix it.
![image](https://user-images.githubusercontent.com/81932478/190577946-287155f2-dd95-470e-96bf-03af4d06de39.png)

Reply:
Hi @Munsa1,
It is because the messages table is empty.
but I just updated the steps to run the app to have this problem.

Reply:
This change has been  done well.